docs/convertToStdVCF.README.md

```diff
@@ -0,0 +1,102 @@
+# Convert DKFZ VCFs to standard-conform VCFs
+
+The VCFs produced by the SNVCallingWorkflow are not standard conform in that some values are not added as additional columns after a single variant column. By contrast, in the [standard](https://samtools.github.io/hts-specs/) format, additional columns should only be used to show variants occurring in additional samples.
```

Comment:
The VCFs generated by the `SNVCallingWorkflow` do not conform to the standard format. This is because annotations are added as additional columns after the variant genotype columns. On the other hand, the [standard](https://samtools.github.io/hts-specs/) format only uses additional columns to display variants occurring in additional samples and adds annotations to the INFO column.